pyproject.toml

```diff
@@ -146,6 +146,8 @@ filterwarnings = [
     "ignore:.*Deprecated call to.*pkg_resources.declare_namespace.*:DeprecationWarning",
     # DeprecationWarning: jax.tree_map is deprecated: use jax.tree.map (jax v0.4.25 or newer) or jax.tree_util.tree_map (any JAX version).
     "ignore:.*jax.tree_map is deprecated.*:DeprecationWarning",
+    # np.testing.assert_array_equal DeprecationWarning
+    "ignore:.*elementwise comparison failed; this will raise an error in the future.*:DeprecationWarning",
```

Comment:
Where is this error originating from? Can it be fixed by `jnp.broadcast_to` or jnp.expand_dims`? If this will throw an error in the future, we should probably try to fix it.